code/game/machinery/doors/airlock.dm

```diff
@@ -252,10 +242,10 @@ About the new airlock wires panel:
 			src.updateUsrDialog()
 		if(AIRLOCK_WIRE_ELECTRIFY)
 			//Cutting this wire electrifies the door, so that the next person to touch the door without insulated gloves gets electrocuted.
-			if(src.secondsElectrified != -1)
+			if(src.electrified != -1)
```

Comment:
```suggestion
			if(electrified != -1)
```

You can get rid of `src` here, since there's no ambiguity `electrified` refers to the `src` var (same for any cases the `src.var` isn't redefined in the proc beforehand, including arguments).